Data validator: relax requirement that p_sigma and q_sigma cannot mix normal and infinite values (#1081)
PowerGridModel/power-grid-model: docs/user_manual/components.md

|               |           |           | &#10060; |

```{note}
1. If both `p_sigma` and `q_sigma` are provided (i.e., not NaN), they represent the standard deviation of the active and
reactive power, respectively, and the value of `power_sigma` is ignored. Any infinite component disables the entire
measurement.

2. If neither `p_sigma` nor `q_sigma` are provided, `power_sigma` represents the standard deviation of the apparent

> power. If it is infinite, it disables the entire measurement.

Reviewer: ```suggestion
power. In this case, infinite value of `power_sigma` disables the entire measurement.
```
